### PR TITLE
[BUG] Update Hosts page for new navigation UI

### PR DESCRIPTION
**Pull request by joepeeples**
Addresses #903. Turns out all screenshots needed to be updated to the new navigation UI, plus some UI references in the text.

Preview [here](https://security-docs_948.docs-preview.app.elstc.co/guide/en/security/master/hosts-overview.html).

### CODE REVIEW

File: docs/management/hosts/hosts-overview.asciidoc [hunk 1]
```diff
@@ -18,30 +18,30 @@ NOTE: The default time range is within the last 15 minutes.
...
+You can add any highlighted value in the data tables to a Timeline query by dragging and dropping the value from the table to the *Timeline* bar at the bottom of the page. You can also select the *Add to timeline investigtation* inline option if you hover over a highlighted value in the table. If the Timeline view is already open, drag and drop the value to the *Query* box. Repeat to add additional fields to the query, or click *+Add Field* in the Timeline view.
 
 [role="screenshot"]
 image::images/drop-to-timeline.png[Drag and drop a value from the data table to the Timeline tab to add it to the query]
 
 *Host detail pages*
 
-Host detail pages display all relative information for a selected host, such as the Host ID, First Seen and Last Seen timestamps, IP addresses, Operating System, and more. To view a host detail page, click on the relative *Host name* link from the *Hosts* tab.
+Host detail pages display all relevant information for a selected host, such as the Host ID, First Seen and Last Seen timestamps, IP addresses, Operating System, and more. To view a host detail page, click on the relevant *Host name* link from the *Hosts* page.
```
Comment (nastasha-solomon) [inline docs/management/hosts/hosts-overview.asciidoc:44]:
```suggestion
Host detail pages display all relevant information for a selected host, such as the Host ID, First seen and Last seen timestamps, IP addresses, Operating system, and more. To view a host detail page, click on the relevant *Host name* link from the *Hosts* page.
```

Comment (nastasha-solomon) [reply docs/management/hosts/hosts-overview.asciidoc:44]: Recommending these changes as they're written this way in the UI.

**Review by nastasha-solomon — APPROVED** [review]
Only have one minor comment. Otherwise, LGTM!

**Review by karanbirsingh-qasource — APPROVED** [review]
required changes are now available in the doc.

**Review by jmikell821 — APPROVED** [review]
LGTM - OK to merge/backport!

**Pull request merged**